### PR TITLE
Added Raider's manifest file

### PR DESCRIPTION
**Pull request by ADBeveridge**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by hfiguiere** [comment]
As I mentionned in the previous PR, changing the app-id also needs to be done upstream:

https://github.com/ADBeveridge/raider/blob/v0.1.0/src/raider.c#L109

File: com.github.ADBeveridge.Raider [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "com.github.ADBeveridge.Raider",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "raider",
```
Comment (hfiguiere) [inline com.github.ADBeveridge.Raider:6]:
```suggestion
    "command" : "raider",
    "rename-icon": "org.gnome.Raider",
    "rename-appdata-file": "org.gnome.Raider.appdata.xml",
    "rename-desktop-file": "org.gnome.Raider.desktop",
```

Also the appdata will not validate, and need the app-id changed.

File: com.github.ADBeveridge.Raider [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "com.github.ADBeveridge.Raider",
```
Comment (hfiguiere) [inline com.github.ADBeveridge.Raider:2]: this file has an incorrect name. It should have the `.json` extension

File: com.github.ADBeveridge.Raider [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "com.github.ADBeveridge.Raider.json",
```
Comment (hfiguiere) [inline com.github.ADBeveridge.Raider:2]: this is not what I mean. The **file** is improperly named.
```suggestion
    "app-id" : "com.github.ADBeveridge.Raider",
```

**Comment by hfiguiere** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by hfiguiere** [comment]
The error is because the text in `<description>` shall at least be encosed in `<p></p>` tag.

**Comment by ADBeveridge** [comment]
OK, I added the <p> tag.

**Comment by hfiguiere** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by hfiguiere** [comment]
bot, build

**Comment by hfiguiere** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by hfiguiere** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by hfiguiere** [comment]
bot, build com.github.ADBeveridge.Raider

File: com.github.ADBeveridge.Raider.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "com.github.ADBeveridge.Raider",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "raider",
+    "rename-icon": "com.github.ADBeveridge.Raider",
+    "rename-appdata-file": "com.github.ADBeveridge.Raider.appdata.xml",
+    "rename-desktop-file": "com.github.ADBeveridge.Raider.desktop",
```
Comment (hfiguiere) [inline com.github.ADBeveridge.Raider.json:9]: these should be the original name. If the name is already correct, the commands should be removed.

(this is why it's failing)

**Comment by ADBeveridge** [comment]
Ok, I removed the rename statements.

**Comment by hfiguiere** [comment]
bot, build com.github.ADBeveridge.Raider

File: com.github.ADBeveridge.Raider.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "com.github.ADBeveridge.Raider",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "raider",,
```
Comment (hfiguiere) [inline com.github.ADBeveridge.Raider.json:6]:
```suggestion
    "command" : "raider",
```

**Comment by ADBeveridge** [comment]
Ok, I got rid of the commas.

**Comment by hfiguiere** [comment]
btw you can also trigger the build yourself.

**Comment by hfiguiere** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by hfiguiere** [comment]
Build fail because the appdata file is missing some fields.

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Comment by ADBeveridge** [comment]
bot, build com.github.ADBeveridge.Raider

**Pull request closed without merging**